### PR TITLE
v2.5.3-beta1

### PR DESCRIPTION
**Pull request by rafeca**

### CODE REVIEW

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,13 @@
 {
   "releases": {
+    "2.5.3-beta2": [
+      "[Added] Stash changes without switching branches - #10032",
+      "[Fixed] Support launching Desktop from WSL on Windows - #4998. Thanks @shawnfunke!",
+      "[Fixed] Set initial value of checkbox to not show again the Discard line confirmation dialog to false - #9989",
```
Comment (billygriffin) [inline changelog.json:6]:
```suggestion
      "[Fixed] Set initial value of opt out checkbox in Discard line confirmation dialog to false - #9989",
```

File: changelog.json [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
+      "[Fixed] Select correct Branch to Compare when two branches have similar names - #9977",
```
Comment (billygriffin) [inline changelog.json:7]:
```suggestion
      "[Fixed] Select correct branch to compare when two branches have similar names - #9977",
```

File: changelog.json [hunk 3]
```diff
@@ -1,5 +1,13 @@
...
+      "[Improved] Add detection to disable username and password authentication when GitHub.com deprecates it - #9231",
```
Comment (billygriffin) [inline changelog.json:8]:
```suggestion
      "[Improved] Add detection to disable username and password authentication when the API is no longer available - #9231",
```

**Comment by ruggi99** [comment]
Isn't the title "2.5.3-beta2"?

**Comment by rafeca** [comment]
> Isn't the title "2.5.3-beta2"?

Yup you're right, just updated the title!

**Pull request merged**